### PR TITLE
Feature: Fixes

### PR DESCRIPTION
**Pull request by MartynasZilinskas**

### CODE REVIEW

File: packages/ts-docs-gen/package.json [hunk 1]
```diff
@@ -15,21 +15,21 @@
   "engine": "node >= 7.5.0",
   "author": "simplrjs <simplr@quatrodev.com> (https://github.com/simplrjs)",
   "dependencies": {
-    "@simplrjs/markdown": "^1.0.8",
+    "@simplrjs/markdown": "^1.1.0",
     "@types/fs-extra": "^5.0.0",
-    "fast-glob": "^1.0.1",
+    "fast-glob": "^2.0.1",
```
Comment (DeividasBakanas) [inline packages/ts-docs-gen/package.json]: Is this package is used somewhere in the project?

File: packages/ts-docs-gen/src/api-items/api-definition-base.ts [hunk 2]
```diff
@@ -54,22 +54,9 @@ export abstract class ApiDefinitionBase<TKind extends Contracts.ApiBaseItemDto =
...
-        return Helpers.Flatten(
-            members.map(x => x.ToText(render).map(y => `${GeneratorHelpers.Tab(tab)}${y}`))
-        );
-    }
-
     protected SerializedTypeToString(render: ReferenceRenderHandler, apiType: ApiTypes | undefined): string {
         if (apiType == null) {
-            // TODO: Add Log for missing type.
+            GeneratorHelpers.LogWithApiItemPosition(LogLevel.Warning, this.ApiItem, "Missing type.");
             return "???";
```
Comment (DeividasBakanas) [inline packages/ts-docs-gen/src/api-items/api-definition-base.ts]: Maybe we should change `???` to some explicit string for example `MISSING_TYPE` and describe it in documentation?

**Review by DonatasBrazionis — APPROVED** [review]

**Review by DeividasBakanas — APPROVED** [review]

File: packages/ts-docs-gen/package.json (hunk 1 above)
Comment (MartynasZilinskas) [reply packages/ts-docs-gen/package.json]: It was a leftover from test generating. I removed it now.

File: packages/ts-docs-gen/src/api-items/api-definition-base.ts (hunk 2 above)
Comment (MartynasZilinskas) [reply packages/ts-docs-gen/src/api-items/api-definition-base.ts]: It goes with an error. I will create an issue to include about this in documentation. Changed from `???` to `Contracts.TypeKeywords.Unknown`.

**Pull request merged**